argoverse/visualization/vis_mask.py

```diff
@@ -37,17 +37,23 @@
 Segment = Tuple[float, float, float, float]
 
 
-def vis_mask(image: np.ndarray, mask: np.ndarray, color: float, alpha: float = 0.4) -> np.ndarray:
-    """Visualize a single binary mask.
+def vis_mask(
+    image: np.ndarray, mask: np.ndarray, color: Union[float, Tuple[int, int, int]], alpha: float = 0.4
```

Comment:
`color: Union[float, Tuple[int, int, int]]` -- does a single `float` represent greyscale?

Reply:
that's right